modules/oe_list_pages_open_vocabularies/src/SearchApiConfigurator.php

```diff
@@ -133,7 +143,11 @@ public function updateConfig(OpenVocabularyAssociationInterface $association, st
     $facet->setFacetSourceId($list_source->getSearchId());
     $facet->setWidget('oe_list_pages_multiselect', []);
     $facet->setFieldIdentifier($property_path);
-    $facet->save();
+
```

Comment:
This event is also thrown if the vocabulary association is being edited so we should either rename it or make it only to be fired for new associations being saved.